### DISCUSSION TITLE
Is there a way to make vertical scroll bar wider?

### DISCUSSION THREAD

**Discussion by ctrl-alt-d** [AvaloniaCommunity/Material.Avalonia#178] [Q&A] [answered]
My Material Avalonia app uses vertical scroll bars in some controls:

<img width="1227" alt="image" src="https://user-images.githubusercontent.com/3105983/177035841-18725757-67b9-4405-8622-4dc89e1cf31c.png">

The [window scroll bar code](https://github.com/ctrl-alt-d/eaprecull/blob/fd3968f20fe226b7e0e308cebadd4527234bac01/UI.ER.AvaloniaUI/Pages/AlumneUpdateWindow.axaml#L14-L26
):

```xml
    <ScrollViewer>
        <StackPanel Orientation="Vertical"
                    Margin="16, 16">
            <Grid>
                <Grid.RowDefinitions>
                    <RowDefinition Height="Auto" />
                    <RowDefinition Height="Auto" />
                    <RowDefinition Height="Auto" />
```

The [list](https://github.com/ctrl-alt-d/eaprecull/blob/fd3968f20fe226b7e0e308cebadd4527234bac01/UI.ER.AvaloniaUI/Pages/AlumneSetWindow.axaml#L112-L121
) code:

```xml
        <ListBox DockPanel.Dock="Bottom"
                 Margin="0"
                 Items="{Binding MyItems, Mode=OneWay}"
                 VirtualizationMode="Simple">
            <ListBox.ItemTemplate>
                <DataTemplate>
                    <pages:AlumneRowUserCtrl/>
                </DataTemplate>
            </ListBox.ItemTemplate>
        </ListBox>
```
There is some way to make scroll bar control wider? My users say they have problems on windows some time to click on it.

Thanks!

**Comment by SKProCH** [comment, marked as answer]
Hello, @ctrl-alt-d 

You can set `Classes="Classic"` for your ScrollViewer and your scrollbar will be look like that:
![image](https://user-images.githubusercontent.com/29896317/177036395-bc5a37b2-73e4-4ddf-9222-4d17ed4710b8.png)
